File: app/src/main/java/com/expansion/lg/kimaru/training/objs/TrainingClass.java
package com.expansion.lg.kimaru.training.objs;

/**
 * Created by kimaru on 2/21/18.
 */

public class TrainingClass {

    Integer id,createdBy;
    String trainingId, className, country, dateCreated, comment;
    Long clientTime;
    boolean archived;

    public TrainingClass() {
    }

    public TrainingClass(Integer id, Integer createdBy, String trainingId,
                         String className, String country, String dateCreated,
                         String comment, Long clientTime, boolean archived) {
        this.id = id;
        this.createdBy = createdBy;
        this.trainingId = trainingId;
        this.className = className;
        this.country = country;
        this.dateCreated = dateCreated;
        this.comment = comment;
        this.clientTime = clientTime;
        this.archived = archived;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Integer getCreatedBy() {
        return createdBy;
    }

    public void setCreatedBy(Integer createdBy) {
        this.createdBy = createdBy;
    }

    public String getTrainingId() {
        return trainingId;
    }

    public void setTrainingId(String trainingId) {
        this.trainingId = trainingId;
    }

    public String getClassName() {
        return className;
    }

    public void setClassName(String className) {
        this.className = className;
    }

    public String getCountry() {
        return country;
    }

    public void setCountry(String country) {
        this.country = country;
    }

    public String getDateCreated() {
        return dateCreated;
    }

    public void setDateCreated(String dateCreated) {
        this.dateCreated = dateCreated;
    }

    public String getComment() {
        return comment;
    }

    public void setComment(String comment) {
        this.comment = comment;
    }

    public Long getClientTime() {
        return clientTime;
    }

    public void setClientTime(Long clientTime) {
        this.clientTime = clientTime;
    }

    public boolean isArchived() {
        return archived;
    }

    public void setArchived(boolean archived) {
        this.archived = archived;
    }
}

File: app/src/main/java/com/expansion/lg/kimaru/training/objs/TrainingClass.kt
package com.expansion.lg.kimaru.training.objs

/**
 * Created by kimaru on 2/21/18.
 */

class TrainingClass {

    var id: Int? = null
    var createdBy: Int? = null
    var trainingId: String
    var className: String
    var country: String
    var dateCreated: String
    var comment: String
    var clientTime: Long? = null
    var isArchived: Boolean = false
    var color = -1

    constructor() {}

    constructor(id: Int?, createdBy: Int?, trainingId: String,
                className: String, country: String, dateCreated: String,
                comment: String, clientTime: Long?, archived: Boolean) {
        this.id = id
        this.createdBy = createdBy
        this.trainingId = trainingId
        this.className = className
        this.country = country
        this.dateCreated = dateCreated
        this.comment = comment
        this.clientTime = clientTime
        this.isArchived = archived
    }
}
